onboardingtaskapp.client/src/components/Sale/Create.jsx

```diff
@@ -20,27 +20,58 @@ export default function Create() {
     const [stores, setStores] = useState([]);
 
     useEffect(() => {
-        fetch('https://localhost:7207/api/Customer')
-            .then(response => response.json())
-            .then(
-                data => setCustomers(data)                
-            )
-            .catch(error => console.error('Error fetching customers:', error));
+        fetchCustomers()
+        fetchProducts()
+        fetchStores()
+       
     }, []);      
 
-    useEffect(() => {
-        fetch('https://localhost:7207/api/Product')
-            .then(response => response.json())
-            .then(data => setProducts(data))
-            .catch(error => console.error('Error fetching products:', error));
-    }, []);
-   
-    useEffect(() => {
-        fetch('https://localhost:7207/api/Store')
-            .then(response => response.json())
-            .then(data => setStores(data))
-            .catch(error => console.error('Error fetching stores:', error));
-    }, []);
+    // const
+    const API_END_POINT = `https://onboardsite.azurewebsites.net/api/`;
```

Comment:
1. Please check my previous comment about console.logs
2. Please implement the API endpoint that I mentioned above